stage/user.js

```diff
@@ -0,0 +1,41 @@
+const MSG = require("./messages");
+const { client } = require("../config");
+
+module.exports = class User {
+  static fromMention(mention) {
+    //return discord user from mention
+    if (!mention) return;
+
+    const matches = mention.match(/^<@!?(\d+)>$/);
+    if (!matches) return;
+
+    const id = matches[1];
+    return client.users.cache.get(id);
+  }
+
+  static getUserWanted(user1, user2, tour, levelsRemaining) {
+    //return user whose answer is awaited
+    if (tour == 1) {
+      if (levelsRemaining > 6) return user1;
+      else if (levelsRemaining > 2) return user2;
+      else return user1;
```

Comment:
Personally I'd extract all this logic about turns / levelsRemaining into a separate file, which would in essence contain your core logic.